### ISSUE TITLE
Implement new "dsl_ode_action_bbox_format_new" service

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#567]
```C
/**
 * @brief Creates a uniquely named "Format Bounding Box" ODE Action that updates 
 * an Object's RGBA bounding box line width and color. 
 * Note: setting the line width to 0 will exclude/hide the object's bounding box from view.
 * @param[in] name unique name for the "Format Bounding Box" ODE Action. 
 * @param[in] border_width border line width for the object's bounding box. 
 * Set to 0 to exclude/hide the border from view.
 * @param[in] border_color RGBA Color for the bounding box.
 * @param[in] has_bg_color set to true to enable background color, false otherwise
 * @param[in] bg_color RGBA Color for the Text background if set
 * @return DSL_RESULT_SUCCESS on success, one of DSL_RESULT_ODE_ACTION_RESULT otherwise.
 */
DslReturnType dsl_ode_action_bbox_format_new(const wchar_t* name, uint border_width, 
    const wchar_t* border_color, boolean has_bg_color, const wchar_t* bg_color);

```

**Comment by rjhowell44** [comment]
Merged into v0.19.alpha

**Comment by rjhowell44** [comment]
Docs are done
